tensorrt_llm/bench/build/tuning.py

```diff
@@ -148,13 +170,16 @@ def finetune_setting(
     raw_token = min(raw_bs * (1 + input_len / output_len), 32768)
 
     # Fine-tune the max batch size.
-    # Set min BS to be 64.
-    if raw_bs < 256:
-        max_bs = max(64, 32 * math.ceil(raw_bs / 32))
-    elif raw_bs < 1024:
-        max_bs = 128 * math.ceil(raw_bs / 128)
+    if enable_optimistic_tuning:
+        # Set min BS to be 64.
+        if raw_bs < 256:
+            max_bs = max(64, 32 * math.ceil(raw_bs / 32))
+        elif raw_bs < 1024:
+            max_bs = 128 * math.ceil(raw_bs / 128)
+        else:
+            max_bs = 256 * math.ceil(raw_bs / 256)
     else:
-        max_bs = 256 * math.ceil(raw_bs / 256)
+        max_bs = 2 * math.floor(raw_bs / 2)
```

Comment:
How was this math determined? Are we saying that this approximation holds in general if we are trying to be less optimistic across all models?

Reply:
Yeah.. this is a just a floor round to the nearest even number. My thought process was that if we don't want to be optimistic (i.e. increase the max batch size), we can just keep it as is. I rounded it down to an even number just because even numbers are nicer.

Reply:
It works for Nemotron-H :) (the only relevant hybrid model)